src/develop/develop.c

```diff
@@ -1508,15 +1509,12 @@ static gboolean _dev_auto_apply_presets(dt_develop_t *dev)
   const char *workflow = dt_conf_get_string_const("plugins/darkroom/workflow");
 
   const gboolean auto_apply_filmic =
-    is_raw
-    && strcmp(workflow, "scene-referred (filmic)") == 0;
+    (is_raw || is_mono) && (strcmp(workflow, "scene-referred (filmic)") == 0);
   const gboolean auto_apply_sigmoid =
-    is_raw
-    && strcmp(workflow, "scene-referred (sigmoid)") == 0;
-
+    (is_raw || is_mono) && (strcmp(workflow, "scene-referred (sigmoid)") == 0);
   const gboolean auto_apply_cat = has_matrix && is_modern_chroma;
 
-  if(auto_apply_filmic || auto_apply_cat)
+  if(auto_apply_filmic || auto_apply_sigmoid || auto_apply_cat)
```

Comment:
Outch! This is a clear bug... well chance was that sigmoid was also for scene-referred and so cat what applied, that's why it was working "by chance".

Reply:
yeah, found that by luck :-)